notebooks/CustomerScenarios/Case Study - Customer Segmentation at An Online Media Company.ipynb

```diff
@@ -0,0 +1,856 @@
+      "text/plain": [
+       "<Figure size 720x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+   
```

Comment:
```suggestion
    "From the plot above, it's clear to see that the true treatment effect is a **nonlinear** function of income, with elasticity around -1.75 when income is smaller than 1 and a small negative value when income is larger than 1. The model fits a quadratic treatment effect, which is not a great fit. But it still captures the overall trend: the elasticity is negative and people are less sensitive to the price change if they have higher income."
```

Reply:
I don't know why this and the following suggestions are targeted at the wrong lines.  Please don't commit them without verifying that they can be applied to the right place...